bundle.go

```diff
@@ -112,6 +120,25 @@ func downloadBundle(dep DataDeployment) {
 	deploymentsChanged <- dep.ID
 }
 
+func (r *DownloadRequest) checkTimeout() {
+
+	if !r.timeoutAfter.IsZero() {
+		if time.Now().After(r.timeoutAfter) {
+			r.timeoutAfter = time.Time{}
+			log.Debugf("bundle download timeout. marking deployment %s failed. will keep retrying: %s",
+				r.dep.ID, r.dep.BundleURI)
+			setDeploymentResults(apiDeploymentResults{
```

Comment:
It's never checked since there error will already be logged and I didn't feel there was anything further that could be done about it at this point.